sentences/fr/_common.yaml

```diff
@@ -101,7 +101,54 @@ responses:
       {%- endif %}
     no_entity: "Désolé, je ne connais pas l'appareil {{ entity }}"
     no_entity_in_area: "Désolé, je ne connais pas l'appareil {{ entity }}"
-
+    no_floor: "Désolé, je ne connais pas l'étage {{ floor }}"
+    no_domain_in_floor: |
+      {% set translations = {
+        "button": "de boutons",
+        "camera": "de caméras",
+        "input_button": "de boutons",
+        "alarm_control_panel": "d'alarmes",
+        "automation": "d'automatisations",
+        "fan": "de ventilateurs",
+        "climate": "de thermostats",
+        "humidifier": "d'humidificateurs",
+        "input_boolean": "de commutateurs",
+        "siren": "de sirènes",
+        "water_heater": "de ballon d'eau chaude",
+        "light": "de lumières",
+        "switch": "de commutateurs",
+        "script": "de scripts",
+        "remote": "de télécommandes",
+        "lock": "de verrous",
+        "vacuum": "d'aspirateurs",
+        "scene": "de scènes",
+        "media_player": "de lecteurs multimédia",
+        "lawn_mower": "de tondeuses à gazon",
+        "valve": "de vannes"
+        } %}
+      {% if domain in translations -%}
+        Désolé, je n'ai pas trouvé {{ translations[domain] }} dans cet étage
+      {%- else -%}
+        Désolé, je n'ai rien trouvé de correspondant dans cet étage
+      {%- endif %}
+    no_device_class_in_floor: |
+      {% set translations = {
+        "awning": "d'auvents",
+        "blind": "de stores",
+        "curtain": "de rideaux",
+        "door": "de portes",
+        "garage": "de portes de garage",
+        "gate": "de portes",
```

Comment:
Why translate "gate" as "porte" instead of "portail" ?

Reply:
You're right, it should be "portail" 👍